packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1681,26 +1693,40 @@ export class Grapher
         return new Bounds(0, 0, DEFAULT_GRAPHER_WIDTH, DEFAULT_GRAPHER_HEIGHT)
     }
 
+    @computed get portraitBounds(): Bounds {
+        return new Bounds(0, 0, DEFAULT_GRAPHER_HEIGHT, DEFAULT_GRAPHER_WIDTH)
```

Comment:
I'm doing a simple swap for now, but will look into what's a sensible default in subsequent PRs